app/javascript/controllers/dropzone_controller.js

```diff
@@ -41,6 +41,10 @@ export default class extends Controller {
     const filepaths = fileNameNodes.map(fileNameNode => {      
         const path = fileNameNode.getAttribute('data-dropzone-path')
         const filename = fileNameNode.innerText.trim()
+
+        const item = fileNameNode.closest('.dz-complete')
+        if(item && item.querySelector("input[name*='_destroy']").value == 1) return null
```

Comment:
don't we usually see these guard clauses as 

```
return null if condition
``` 

?

Reply:
In Ruby, but not in JS.

Reply:
oh duh.

Reply:
To be honest, I initially coded it as `return if condition`.